src/qt/transactionview.cpp

```diff
@@ -145,36 +145,22 @@ TransactionView::TransactionView(QWidget* parent) :
 
     transactionView->setObjectName("transactionView");
 
-    // Actions
-    abandonAction = new QAction(tr("Abandon transaction"), this);
-    resendAction = new QAction(tr("Resend transaction"), this);
-    copyAddressAction = new QAction(tr("Copy address"), this);
-    copyLabelAction = new QAction(tr("Copy label"), this);
-    QAction *copyAmountAction = new QAction(tr("Copy amount"), this);
-    QAction *copyTxIDAction = new QAction(tr("Copy transaction ID"), this);
-    QAction *copyTxHexAction = new QAction(tr("Copy raw transaction"), this);
-    QAction *copyTxPlainText = new QAction(tr("Copy full transaction details"), this);
-    QAction *editLabelAction = new QAction(tr("Edit address label"), this);
-    QAction *showDetailsAction = new QAction(tr("Show transaction details"), this);
-    QAction *showAddressQRCodeAction = new QAction(tr("Show address QR code"), this);
-#ifndef USE_QRCODE
-    showAddressQRCodeAction->setEnabled(false);
-#endif
-
     contextMenu = new QMenu(this);
     contextMenu->setObjectName("contextMenu");
-    contextMenu->addAction(copyAddressAction);
-    contextMenu->addAction(copyLabelAction);
-    contextMenu->addAction(copyAmountAction);
-    contextMenu->addAction(copyTxIDAction);
-    contextMenu->addAction(copyTxHexAction);
-    contextMenu->addAction(copyTxPlainText);
-    contextMenu->addAction(showDetailsAction);
-    contextMenu->addAction(showAddressQRCodeAction);
+    copyAddressAction = contextMenu->addAction(tr("Copy address"), this, &TransactionView::copyAddress);
+    copyLabelAction = contextMenu->addAction(tr("Copy label"), this, &TransactionView::copyLabel);
+    contextMenu->addAction(tr("Copy amount"), this, &TransactionView::copyAmount);
+    contextMenu->addAction(tr("Copy transaction ID"), this, &TransactionView::copyTxID);
+    contextMenu->addAction(tr("Copy raw transaction"), this, &TransactionView::copyTxHex);
+    contextMenu->addAction(tr("Copy full transaction details"), this, &TransactionView::copyTxPlainText);
+    contextMenu->addAction(tr("Show transaction details"), this, &TransactionView::showDetails);
     contextMenu->addSeparator();
-    contextMenu->addAction(abandonAction);
-    contextMenu->addAction(resendAction);
-    contextMenu->addAction(editLabelAction);
+    abandonAction = contextMenu->addAction(tr("Abandon transaction"), this, &TransactionView::abandonTx);
```

Comment:
gui263:
```suggestion
    abandonAction = contextMenu->addAction(tr("Abandon transaction"), this, &TransactionView::abandonTx);
    resendAction = contextMenu->addAction(tr("Resend transaction"), this, &TransactionView::resendTx);
```

Reply:
was this resolved?

Reply:
yes: https://github.com/dashpay/dash/pull/6168/commits/4f89c98dc40bf77ac8a9dffe75ed0d117ef91184#diff-dab82be063a09e257636f19825b81c93782c2e3158a4d471a45208a0568edb7cR159